### PR TITLE
Add Rubocop rule for clevel constants

### PR DESCRIPTION
**Pull request by aditya-stripe**
Moving here from pay-server.

r? @pt-stripe 
cc @stripe/observability

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,4 +5,5 @@
 end || source('https://rubygems.org/')
 
 gemspec
+gem 'stripe-rubocop', '0.4.4'
```
Comment (pt-stripe) [inline Gemfile]: instead of relying on this, you should rely on the open source `rubocop` gem

Comment (aditya-stripe) [inline Gemfile]: Good catch - thanks

File: Gemfile [hunk 2]
```diff
@@ -5,4 +5,5 @@
 end || source('https://rubygems.org/')
 
 gemspec
+gem 'rubocop', '0.48.1'
```
Comment (pt-stripe) [inline Gemfile]: how about `>= 0.48.1` so we can upgrade without breaking this

**Comment by pt-stripe** [comment]
lgtm
once you make the version be a base rev

**Comment by aditya-stripe** [comment]
thanks!

self-approving version bump

**Pull request merged**